x/data/server/features/query_attestors.feature

```diff
@@ -0,0 +1,206 @@
+Feature: QueryAttestors
+
+  Background:
+    Given the content entry
+    """
+    {
+      "iri": "regen:13toVfvC2YxrrfSXWB5h2BGHiXZURsKxWUz72uDRDSPMCrYPguGUXSC.rdf",
+      "timestamp": "2020-01-01T00:00:00.000000000Z"
+    }
+    """
+
+  Rule: one attestor is returned if one attestor entry exists
```

Comment:
I don't think this is an accurate use of the Rule keyword but it would be nice to group cases together that use the same Background. Should this just be a scenario with sub-scenarios?